CONTRIBUTING.md

```diff
@@ -123,11 +129,21 @@ See the next section for how to adapt to upstream changes.
 
 ### Branching Strategy
 
-By default, development happens in branches, which are merged via pull requests.
+By default, development happens in branches, which are merged via pull requests (PRs from `master` will be closed).
```

Comment:
This is just some feedback from my experience contributing here:

I ran into that when I created my first pull request. 
This probably occured because my knowledge on git and github is not too deep. I only github forked the junit-pioneer repo and thought that this was also a kind of branch and created a pull request form my/junit-pioneer/master to the junit-pioneer/junit-pioneer/master.

Maybe a hint for non maintainers that they have to github fork and create a new branch (is this even correct?) can prevent noobs like me for doing this mistake. 

I understand that people need to have certain amount of  basic knowledge before you they can contribute and that this document should not teach the basics of git/github but at the same time lower the threshold for people to contribute.

Reply:
Thanks for letting us know. :+1: I opened #243.

Reply:
Thanks @Hancho2009 , will add my thoughts in #243